app/graphql/mutations/create_video.rb

```diff
@@ -0,0 +1,28 @@
+module Mutations
+  class CreateVideo < BaseMutation
+    argument :vimeo_id, Integer, required: true
+    argument :speaker, String, required: false
+    argument :topic, String, required: false
+    argument :date_recorded, String, required: false
+
+    type Types::VideoType
+
+    def resolve(vimeo_id: nil, speaker: nil, topic: nil, date_recorded: nil)
+      video = VideoFacade.new(vimeo_id)
```

Comment:
I would name this local variable something like `video_facade` to avoid conflating this object with the model instance created on line 12+.